src/CMakeLists.txt

```diff
@@ -462,6 +464,18 @@ if( MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "HIP")
         )
 endif()
 
+list(APPEND MIOpen_Source
+    cellfft/include/cellfft_get_kernel.hpp
+    cellfft/include/cellfft_op.hpp
+    cellfft/include/cellfft_param.hpp
+    cellfft/source/cellfft_get_kernel.cpp
+    cellfft/source/cellfft_op.cpp
+    cellfft/source/cellfft_param.cpp
+    include/miopen/conv/invokers/cellfft.hpp
+    conv/invokers/cellfft.cpp
+    solver/conv_cellfft.cpp
```

Comment:
Invokers are used in one module only, i.e. in the cellfft solver. There is no need to have separate module for invoker, it can be included directly into the solver. Likewise, all other modules and respective interfaces (.cpp and .hpp files). AFAICS all host code can be moved into solver (and made that code non-global). This would open path for strong compiler optimizations, for example and cleans global namespace.

Reply:
This is not resolved.